lrc/Pos/Lrc/Genesis.hs

```diff
@@ -2,11 +2,13 @@ module Pos.Lrc.Genesis
        ( genesisSeed
        ) where
 
-import           Pos.Core.Types (SharedSeed (..))
+import           Pos.Core.Genesis.Parser (genesisSpec)
+import           Pos.Core.Genesis.Types  (gsFtsSeed)
+import           Pos.Core.Types          (SharedSeed (..))
 
 -- | Seed that will be used for the 0th epoch. We must hardcode a seed
 -- because we need to somehow determine leaders for the first ever epoch
 -- (stakes are hardcoded as well so we can run FTS on those stakes using this
 -- seed).
 genesisSeed :: SharedSeed
-genesisSeed = SharedSeed "vasa opasa skovoroda Ggurda boroda provoda"
```

Comment:
now, it didn't rip. `ftsSeed: "c2tvdm9yb2RhIEdndXJkYSBib3JvZGEgcHJvdm9kYSA="` this string in base64 equals this legendary seed. @gromakovsky controlled it would be left in the code. he encoded it in `base16`, `base64`, `base58` and etc for all possible cases. I hope he wrote a test which validates this seed equals this string.